documentation/modules/proc_enabling-nbde-with-clevis.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// assembly_planning-migration-vmware.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc_enabling-nbde-with-clevis_{context}"]
+
+= Enabling Network-Bound Disk Encryption with Clevis 
+
+[role="_abstract"]
+When you enable Network-Bound Disk Encryption (NBDE) with Clevis, the Tang servers automatically decrypt the Linux Unified Key Setup (LUKS)-encrypted disks during a migration. If you do not use NBDE with Clevis, you must manually add passphrases for LUKS-encrypted devices so that the Tang servers can decrypt the disks during a migration. 
+
+You can enable NBDE with Clevis either in the MTV UI or in the YAML file for your migration plan:
+
+* In the MTV UI, you must select either NBDE with Clevis or LUKS passphrases. You can have only one encryption type, and you apply the setting to all VMs in your migration plan.
+* In the YAML file for your migration plan, you can combine encryption types and apply the setting to selected VMs in the YAML file.
+
+.Prerequisites
+* You have raw copy mode enabled in MTV to ensure that the encrypted data is copied bit-for-bit without any modification.
```

Comment:
You don't need the raw copy mode, quite the opossite, raw copy mode will ignore this conversion